app/views/users/_account_set_password.html.erb

```diff
@@ -4,10 +4,14 @@
         Set new password
       </h2>
       <p class="color-base-70">
-        As an alternative to signing in via linked social accounts, you may create a password.
+        As an alternative to signing in via linked social accounts, you may create a password. If you signed up with a social media account, please <a href="<%= new_user_password_path %>">reset your password</a> first.
```

Comment:
we should tell them to reset the password using their primary email, as they might have several :(

Reply:
@rhymes I made the message more explicit now, what do you think?

![image](https://user-images.githubusercontent.com/47985/104675074-53662700-5717-11eb-89eb-ca0e67cf3d80.png)